Sources/NIOHTTP2/HTTP2CommonInboundStreamMultiplexer.swift

```diff
@@ -307,3 +307,11 @@ extension HTTP2CommonInboundStreamMultiplexer {
         }
     }
 }
+
+extension HTTP2CommonInboundStreamMultiplexer {
+    internal func processedFrame(streamID: HTTP2StreamID, size: Int) {
```

Comment:
Hrm: this doesn't really feel like a "common" function at all. Can we avoid making it seem like one?

Reply:
As discussed I've added a comment to explain that this should only be used from the inline multiplexer. Anything more substantial that I can think of either blurs responsibilities or adds a lot of complexity of abstractions.